### PR TITLE
Update EIP-4844: Minor edits

### PR DESCRIPTION
**Pull request by msjenniferelle**
Minor edits for clarification purposes.

### CODE REVIEW

File: EIPS/eip-5065.md [hunk 1]
```diff
@@ -34,14 +34,14 @@ The total gas cost should be the sum of a static cost + address_access_cost + va
 
 
 ## Rationale
-This behavior is already possible by deploying a new contract that does `SELFDESTRUCT (0xFF)` but it is prohibitively expensive. In most scenarios, the contract author only wants to transfer ether rather than transferring control of the execution. ERC20 can be used as a case study for this where most users transfer funds without a post-transfer hook.
+This behavior is already possible by deploying a new contract that does `SELFDESTRUCT (0xFF)` but it is prohibitively expensive. In most scenarios, the contract author only wants to transfer ether rather than transfer control of the execution. ERC20 can be used as a case study for this where most users transfer funds without a post-transfer hook.
```
Comment (xinbenlv) [inline EIPS/eip-5065.md:37]: "transferring" seems grammatically correct, right?

**Review by xinbenlv — APPROVED** [review]
Thanks for the change. LGTM

**Review by lightclient — CHANGES_REQUESTED** [review]
Please open the changes to the other EIP in a separate PR. Thanks!

**Review by fulldecent — APPROVED** [review]

**Comment by xinbenlv** [comment]
Thank @msjenniferelle for drafting this PR. But we generally edit one EIP at one PR, partly because it require approval by their respective owners. 

Could you split them into different PRs? Until then, consider move this PR back to draft and notify us when new PRs are ready for review again? Again, greatly appreciate your editorial contributions!

**Comment by lightclient** [comment]
@xinbenlv please stop bumping stale topics for no reason. I already requested the author only change one EIP at a time so that we could collect author approvals and he did not respond. This was getting close to being closed as stale now the counter will reset and we'll get pinged at least 2 more times about its staleness before it is closed.

**Comment by fulldecent** [comment]
Recommending manual close. It can always be reopened no worries if needed.

**Pull request closed without merging**